platform/marvell-armhf/sonic-platform-nokia/7215/sonic_platform/chassis.py

```diff
@@ -8,6 +8,7 @@
 try:
     import os
     import sys
+    import subprocess
```

Comment:
Seems like we can remove this import since Line34 will do this.
Remove it will also fix the LGTM alert.

Reply:
Fixed